packages/tezos-baking-015-PtLimaPt/tezos-baking-015-PtLimaPt.16.1/opam

```diff
@@ -42,7 +42,7 @@ depends: [
 build: [
   ["rm" "-r" "vendors"]
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name "-j" jobs] { with-test & ocaml:native }
+  ["dune" "runtest" "-p" name "-j" jobs] { ocaml:native }
```

Comment:
I'm sorry but I'm not sure what's "removed" here. If anything you're now running the tests everytime which is worse and not mergeable.

Reply:
Indeed that was niot the way of removing the tests. 
Is the last version ok ? I removed the tests from  ̀tezos-baking-alpha`, `tezos-baking-016` and `tezos-baking-015` which are the packages release in the v16.
If that's not enough I can suggest a seperate PR that do the same thing for Octez v15 ?